src/stactools/nclimgrid/monthly_stac.py

```diff
@@ -165,11 +182,13 @@ def monthly_base_item(year: int, month: int) -> Item:
                 datetime=None,
                 stac_extensions=[])
 
-    # Projection extension
     projection = ProjectionExtension.ext(item, add_if_missing=True)
     projection.epsg = constants.EPSG
     projection.shape = constants.SHAPE
     projection.transform = constants.TRANSFORM
+    # "projection" is epsg:4326, so we reuse the bbox and geometry
+    projection.bbox = constants.WGS84_BBOX
+    projection.geometry = constants.WGS84_GEOMETRY
```

Comment:
```suggestion
    # "projection" is epsg:4326, so we reuse the bbox and geometry
    projection.bbox = constants.WGS84_BBOX
    projection.geometry = constants.WGS84_GEOMETRY
```

See comments on the item JSON files themselves, but I think we can safely remove these as they're duplicate.

Reply:
Yep. Done.